flink-cdc-common/src/main/java/org/apache/flink/cdc/common/types/utils/DataTypeUtils.java

```diff
@@ -134,8 +136,27 @@ public static org.apache.flink.table.types.DataType toFlinkDataType(DataType typ
                         toFlinkDataType(children.get(0)), toFlinkDataType(children.get(1)));
             case ROW:
                 Preconditions.checkState(!CollectionUtil.isNullOrEmpty(children));
-                return org.apache.flink.table.api.DataTypes.ROW(
-                        children.toArray(new org.apache.flink.table.types.DataType[] {}));
+                RowType rowType = (RowType) type;
+                List<org.apache.flink.table.api.DataTypes.Field> fields =
+                        rowType.getFields().stream()
+                                .map(
+                                        dataField ->
+                                                dataField.getDescription() == null
+                                                        ? org.apache.flink.table.api.DataTypes
+                                                                .FIELD(
+                                                                        dataField.getName(),
+                                                                        toFlinkDataType(
+                                                                                dataField
+                                                                                        .getType()))
+                                                        : org.apache.flink.table.api.DataTypes
+                                                                .FIELD(
+                                                                        dataField.getName(),
+                                                                        toFlinkDataType(
+                                                                                dataField
+                                                                                        .getType()),
+                                                                        dataField.getDescription()))
+                                .collect(Collectors.toList());
```

Comment:
the lambda is so ugly, maybe you can define a map function

Reply:
Add a named `toFlinkDataTypeField` method.

Reply:
In my opinion, `toFlinkDataTypeField` is a method of DataField, not a static function in utils.
Hope for a idea from Project member.

Reply:
Yes, this will be better, Fixed.